<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEditComplex">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Generic Event не используются в межкомплексном. Если очень надо — скажите.</string>
     </property>
     <property name="placeholderText">
      <string>Complex</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditDevice">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Device</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEditEvent">
     <property name="placeholderText">
      <string>Event</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEditValue">
     <property name="placeholderText">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButtonSend">
     <property name="text">
      <string>Send</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditValue</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonSend</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditEvent</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonSend</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditDevice</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonSend</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditComplex</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonSend</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
